command/push.go

```diff
@@ -306,6 +371,7 @@ func (c *atlasPushClient) Get(name string) (map[string]interface{}, error) {
 
 	var variables map[string]interface{}
 	if version != nil {
+		// TODO: merge variables and TFVars
 		//variables = version.Variables
```

Comment:
So this is for the "subsequent pushes" use case, where the client fetches the existing variables and prints which ones are going to be overridden. Will chat you about this.